common.lic

```diff
@@ -93,6 +93,16 @@ module DRC
     end
     true
   end
+  
+  def collect(item)
+    case bput("collect #{item}",'You manage to collect a pile', 'The room is too cluttered')
+    when 'The room is too cluttered'
+      fput('kick pile')
```

Comment:
I'm thinking to err on the side of caution regarding dump junk.  We can't guarantee that there aren't people in the room who have things on the ground they don't want to lose.